app/controllers/judges_controller.rb

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2015 Richa Arora and Kate Fulton All Rights Reserved.
+class JudgesController < ApplicationController
+  before_action :set_judge, only: [:update, :destroy]
+  before_filter :authenticate_user!
+  respond_to :html
+
+  def index
+    @judges = Judge.all
+    respond_with(@judges)
+  end
+
+  def new
+    @judge = Judge.new
+    respond_with(@judge)
+  end
+
+  def create
+    if current_user && current_user.admin
+      @judge = Judge.new(judge_params)
+      @judge.name
+      if @judge.save
+        redirect_to judges_path, notice: "Judge successfully added."
+      else
+        redirect_to judges_path, notice: "Judge unable to be added."
+      end
+    else
+      redirect_to judges_path, notice: "Only admin can create Judge."
+    end
+  end
+
+  def edit
+    set_judge
```

Comment:
perhaps you can check if the user is admin, and set all the fields to readonly. otherwise they might do a lot of editing only to see at update that they can't save it.